CMS/settings.py

```diff
@@ -143,6 +145,33 @@
 }
 
 SIMPLE_JWT = {
-    'AUTH_HEADERS': 'JWT',
-    'ACCESS_TOKEN_LIFETIME': timedelta(days=1),
+    'ACCESS_TOKEN_LIFETIME': timedelta(minutes=5),
+    'REFRESH_TOKEN_LIFETIME': timedelta(days=90),
```

Comment:
why 90 days?

Reply:
90 days is just a number, I prefer a longer time frame because of the fact that it's always annoying for users to keep logging in to the site every time they want to use the service. Think of sites like Facebook, how often do you log in? If you don't log in after some time, it will log you out because of the token expiry date, so you will have to authenticate again.  So you can choose the time frame that suits the function. As long as this is not a banking application where you need to login and be logout out at every session, you can choose the time frame as you wish.

Reply:
@tallninja @Hudson-obai What would you guys recommend for the refresh token lifetime ```'REFRESH_TOKEN_LIFETIME': timedelta(days=90),```